docs/csharp/how-to/concatenate-multiple-strings.md

```diff
@@ -57,7 +57,7 @@ combines an array of words by adding a space between each word in the array:
 
 [!code-csharp-interactive[string concatenation using LINQ expressions](../../../samples/snippets/csharp/how-to/strings/Concatenate.cs#6)]  
 
-You can try these samples by looking at the code in our [GitHub repository](https://github.com/dotnet/docs/tree/master/samples/snippets/csharp/how-to/strings). Or you can download the samples [as a zip file](https://github.com/dotnet/docs/tree/master/samples/snippets/csharp/how-to/strings.zip).
+You can try these samples by looking at the code in our [GitHub repository](https://github.com/dotnet/samples/tree/master/snippets/csharp/how-to/strings). Or you can download the samples [as a zip file](https://github.com/dotnet/samples/tree/master/snippets/csharp/how-to/strings.zip).
```

Comment:
nit: If you change tree to raw, the browser just opens the download dialog without displaying the page and requiring the user select the Download button. This applies as well to the other downloadable samples.

Reply:
I check for all the zip downloads.